applications/asset_tracker/prj.conf

```diff
@@ -48,12 +75,24 @@ CONFIG_MAIN_THREAD_PRIORITY=7
 # Heap and stacks
 CONFIG_HEAP_MEM_POOL_SIZE=16384
 CONFIG_MAIN_STACK_SIZE=8192
-CONFIG_GPS_SIM_THREAD_STACK_SIZE=1024
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
-CONFIG_HW_STACK_PROTECTION=y
+
+# Settings
+#CONFIG_FLASH=y
+#CONFIG_FLASH_PAGE_LAYOUT=y
+#CONFIG_FLASH_MAP=y
+#CONFIG_FCB=y
+#CONFIG_SETTINGS=y
+#CONFIG_SETTINGS_FCB=y
+#CONFIG_MPU_ALLOW_FLASH_WRITE=y
 
 # Disable native network stack to save some memory
-CONFIG_NET_IPV4=n
-CONFIG_NET_IPV6=n
-CONFIG_NET_UDP=n
-CONFIG_NET_TCP=n
+#CONFIG_NET_IPV4=n
+#CONFIG_NET_IPV6=n
+#CONFIG_NET_UDP=n
+#CONFIG_NET_TCP=n
```

Comment:
There should be a comment that let's the reader know that these options weren't commented out by accident.